R/functions.R

```diff
@@ -343,3 +366,165 @@ ate = function(Z,Y){
   mean(Y[ind1])-mean(Y[ind2])
 }
 
+#' Define a house structure vector that describes the population in each household, which is useful in the clustered interference
```

Comment:
"Define a house structure vector that describes the population in each household, which is useful in the clustered interference.  In this running example, houses refer to clusters and kids refer to experimental units."